src/pywbem/twisted_client.py

```diff
@@ -300,6 +302,109 @@ def parseResponse(self, xml):
         return [pywbem.tupleparse.parse_instance(x) for x in tt]
 
 
+class OpenEnumerateInstances(WBEMClientFactory):
+    """Factory to produce EnumerateInstances WBEM clients."""
+
+    def __init__(self, creds, classname, namespace='root/cimv2', **kwargs):
+        self.classname = classname
+        self.namespace = namespace
+        self.context = None
+
+        if not kwargs.get('MaxObjectCount'):
+            kwargs['MaxObjectCount'] = DEFAULT_ITER_MAXOBJECTCOUNT
+
+        payload = self.imethodcallPayload(
+            'OpenEnumerateInstances',
+            namespace,
+            ClassName=CIMClassName(classname),
+            **kwargs)
+
+        WBEMClientFactory.__init__(
+            self,
+            creds,
+            operation='MethodCall',
+            method='OpenEnumerateInstances',
+            object=namespace,
+            payload=payload)
+
+    def __repr__(self):
+        return '<%s(/%s:%s) at 0x%x>' % \
+               (self.__class__, self.namespace, self.classname, id(self))
+
+    def parseResponse(self, xml):
+        res = []
+        part_results = {}
+
+        for paramvalue in xml.findall('.//PARAMVALUE'):
+            str_paramvalue = tostring(paramvalue)
+            tuple_paramvalue = pywbem.tupletree.xml_to_tupletree(str_paramvalue)
+            part_results.update(pywbem.tupleparse.parse_paramvalue_to_dict(tuple_paramvalue))
+
+        for x in xml.findall('.//VALUE.INSTANCEWITHPATH'):
+            s = tostring(x)
+            tt = pywbem.tupletree.xml_to_tupletree(s)
+            part_res = pywbem.tupleparse.parse_value_instancewithpath_to_dict(tt)
+            res.append(part_res['VALUE.INSTANCEWITHPATH'])
+
+        part_results.update({'IRETURNVALUE': res})
+        return self._get_rslt_params(part_results)
```

Comment:
_get_rslt_params is a static method but you invoke it as an instance method, as it doesn't use the state, I would recommend to move it outside the class, otherwise, use OpenEnumerateInstances._get_rslt_params definition to invoke the static method, to make the things more clear.